src/nitro/communication/messages/incoming/nux/NewUserExperienceGiftOptions.ts

```diff
@@ -0,0 +1,40 @@
+import { IMessageDataWrapper } from '../../../../../api';
+import { NewUserExperienceGift } from './NewUserExperienceGift';
+
+export class NewUserExperienceGiftOptions
+{
+    private _dayIndex: number;
+    private _stepIndex: number;
+    private _options: NewUserExperienceGift[];
+
+    constructor(wrapper: IMessageDataWrapper)
+    {
+        this._dayIndex = wrapper.readInt();
+        this._stepIndex = wrapper.readInt();
+        this._options = [];
+
+        const _local_2:number = wrapper.readInt();
+        let _local_3:number;
```

Comment:
Initialize this to 0. Js doesn't behave the same as actionscript. Can also rename _local_3 to `index` since its just an index for our loop